pkg/tfvar/tfvar.go

```diff
@@ -178,6 +180,9 @@ func WriteAsTFEResource(w io.Writer, vars []Variable) error {
 		resourceBody.SetAttributeValue("key", cty.StringVal(v.Name))
 		resourceBody.SetAttributeValue("value", v.Value)
 		resourceBody.SetAttributeValue("sensitive", cty.BoolVal(v.Sensitive))
+		if v.Ephemeral {
+			resourceBody.SetAttributeValue("ephemeral", cty.BoolVal(v.Ephemeral))
+		}
```

Comment:
Are you sure we need this? I don't see the attribute in `tfe_variable` documentation: https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/variable